src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -116,7 +116,8 @@ public Object invokeMethod(String name, Object args) {
         FlowNode an;
 
         // TODO: generalize the notion of Step taking over the FlowNode creation.
```

Comment:
FYI, we actually have a JIRA for this TODO now - [JENKINS-38638](https://issues.jenkins-ci.org/browse/JENKINS-36838). Which I probably should assign away from me.